AIDriver.lua

```diff
@@ -1280,6 +1282,15 @@ function AIDriver:checkForHeapBehindMe(tipper)
 	end
 end
 
+function AIDriver:getHeapSizeBehindMe(tipper)
```

Comment:
@MddMBorg Hi there,

A small code improvement would be to consolidate the functions: ` self:checkForHeapBehindMe(tipper), self:getHeapSizeBehindMe(tipper)`into one like this: 
```
`function AIDriver:checkForHeapBehindMe(tipper)
    local dischargeNode = tipper:getCurrentDischargeNode().node
    local offset = -self.settings.loadUnloadOffsetZ:get()
    offset = courseplay:isNodeTurnedWrongWay(self.vehicle,dischargeNode)and -offset or offset
    local startX,startY,startZ = localToWorld(dischargeNode,0,0,offset) ;
    local tempHeightX,tempHeightY,tempHeightZ = localToWorld(dischargeNode,0,0,offset+0.5) 
    local searchWidth = 1    
    local fillType = DensityMapHeightUtil.getFillTypeAtLine(startX,startY,startZ,tempHeightX,tempHeightY,tempHeightZ, searchWidth)
    if fillType == tipper.cp.fillType then
        local fillLevel = DensityMapHeightUtil.getFillLevelAtArea(fillType,startX,startZ,startX,startZ+1,startX+1,startZ)    --should represent an area of 1m^2 starting at the discharge node
        return true,fillLevel
    end
    return false,0
end`
```